secvarctl.spec

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2023 IBM Corp.
+
+Name:		secvarctl
+Version:        1.0
+Release:        1%{?dist}
+Summary:        PowerPC secure variable control
+BuildArch:      ppc64le ppcl64 x86_64
+
+Group:          None
+License:        Apache 2.0
+URL:            https://github.com/open-power/secvarctl
+Source0:        https://github.com/open-power/secvarctl/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+BuildRequires: cmake openssl-devel
+Requires: openssl
+
+%description
+%{name} facilitates manipulation of PowerPC authenticated variables from
+userspace via sysfs.  The primary use case is management of secure boot
+variables.
+
+%global debug_package %{nil}
+
+%prep
+%setup -q
+cmake . -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="/usr"
```

Comment:
This can just be `%cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="/usr"`, later versions of rpmbuild in `el9`, fedora, etc actually set the build output directory, which is expected in the later `%cmake_build` call.

Reply:
Quick update because I just thought to look at the [documentation](https://docs.fedoraproject.org/en-US/packaging-guidelines/CMake/), but it looks like `-DCMAKE_INSTALL_PREFIX` is also set by default by invoking `%cmake`, so only the build type setting is needed. I will test this tomorrow.